robosuite/robots/mobile_robot.py

```diff
@@ -266,6 +266,15 @@ def setup_references(self):
             self.sim.model.joint_name2id(joint) for joint in self.robot_model.head_joints
         ]
 
+        self._ref_actuator_to_joint_id = np.ones(self.sim.model.nu).astype(np.int32) * (-1)
+        for part_name, actuator_ids in self._ref_actuators_indexes_dict.items():
+            self._ref_actuator_to_joint_id[actuator_ids] = np.array(
+                [
+                    self._ref_joints_indexes_dict[part_name].index(self.sim.model.actuator_trnid[i, 0])
+                    for i in actuator_ids
+                ]
+            )
+
```

Comment:
I think we should have the same code in `fixed_base_robot.py` as well, right?

Reply:
Yes.